### ISSUE TITLE
Navigation items incorrectly described by screen readers (critical)

### ISSUE THREAD

**Issue by MelanieHotchkiss** [magda-io/magda#1907]
### Problem description
Intopia issue 4
Main menu navigation items are not announced correctly by screen readers. Neither JAWS/IE nor NVDA/FF announce the navigation items correctly. JAWS announces each menu item as “logo visited link graphic”. NVDA announces the first item as “blank” and ignores the next three items.

### Problem reproduction steps
Investigate - Remove the aria-hidden="true" attribute from the target <div>, and the menu items will be announced correctly.

### Screenshot / Design / File reference
Intopia:
[4] Navigation items incorrectly described by screen readers
Type: WCAG 2.1 failure
Fails 4.1.2 Name, Role, Value - Level A
Location: Navigation menu, header, all pages
Platform: Desktop
intopia.digi tal 16
Severity: Critical
What’s the issue?
Main menu navigation items are not announced correctly by screen readers. Neither JAWS/IE nor NVDA/FF announce the navigation items correctly. JAWS announces each menu item as “logo visited link graphic”. NVDA announces the first item as “blank” and ignores the next three items.
Why it matters
It is critical that navigation items are correctly announced by screen readers in order to enable users to navigate the site.
Our recommendation
Remove the aria-hidden="true" attribute from the target <div>, and the menu items will be announced correctly.
